### PR TITLE
#145 Cite-proc ignores et-al attributes if they set on bibliography/citation level.

### PR DESCRIPTION
**Pull request by xopoc14**

### CODE REVIEW

**Comment by seboettg** [comment]
Hi @xopoc14!
Thank you for your contribution! For some reason citeproc is rendering `…` instead of `&#38;` (ampersand) for various tests. Do you have any idea what could be the reason for this?

**Comment by xopoc14** [comment]
@seboettg I am working on fixing for the broken tests.

**Comment by xopoc14** [comment]
@sebboettg I fixed broken tests and added a test for this bug.

File: src/Rendering/Name/Name.php [hunk 1]
```diff
@@ -245,21 +245,35 @@ private function cloneNamePOSC($name)
...
-    protected function appendEtAl($data, $text, $resultNames)
+    protected function isEtAl($data, $resultNames): bool
     {
-        //append et al abbreviation
         if (count($data) > 1
             && !empty($resultNames)
             && !empty($this->etAl)
             && !empty($this->etAlMin)
             && !empty($this->etAlUseFirst)
             && count($data) != count($resultNames)
```
Comment (seboettg) [inline src/Rendering/Name/Name.php:259]: You may write it shorter:
```
protected function isEtAl($data, $resultNames): bool {
    return count($data) > 1 && 
        !empty($resultNames) && 
        !empty($this->etAl) && 
        !empty($this->etAlMin) && 
        !empty($this->etAlUseFirst) && 
        count($data) != count($resultNames)
}
```

**Review by seboettg — APPROVED** [review]
Great job!

File: src/Rendering/Name/Name.php (hunk 1 above)
Comment (xopoc14) [reply src/Rendering/Name/Name.php]: @seboettg updated

**Pull request merged**